pypeit/images/pypeitimage.py

```diff
@@ -823,4 +815,175 @@ def __repr__(self):
         repr = repr + '>'
         return repr
 
+    def _base_header(self, hdr=None):
```

Comment:
love it.  Am wondering if it should be a `static` method
in case other code wants it.

Reply:
Let's cross that bridge when we come to it.